Makefile

```diff
@@ -33,9 +33,9 @@ install-cdkproxy:
 install-tests:
 	pip install -r tests/requirements.txt
 
-lint:
```

Comment:
I wouldn't makefile target tool specific, let's keep it as `lint`

Reply:
May be we can keep both? I don't want the name to be confusing, since it's just an alias

Reply:
Previously we were running `flake8` and we called the target `lint` and not `flake8` which I think is correct because it's abstracting whatever underlying linter we use and also that way we wouldn't have to change the pipelines code.